src/VCalibrator.cpp

```diff
@@ -4682,10 +4682,13 @@ bool VCalibrator::calculateIPRGraphs( string iPedFileName, unsigned int iSummati
 	float norm = hIPR->Integral( 1, hIPR->GetNbinsX() );
 	if( norm < fPedPerTelescopeTypeMinCnt )  //statistical limit for number of counts
 	{
+		fRunPar->fIPRAverageTel = true;
 		cout << "Telescope " << iTelType << ": ";
 		cout << "VCalibrator::calculateIPRGraphs WARNING: too few statistics to measure IPR curves ";
 		cout << "(total counts available: " << norm << ", ";
 		cout << "current limit " << fPedPerTelescopeTypeMinCnt << ")" << endl;
+		cout << "IPR graphs will be provided as sum of 4 telescopes statistics." << endl;
```

Comment:
Hardwired total number of 4 telescopes (note that there are quite a few 3-telescope runs).

How do we support / warn the user when the run is very short and even the averaging over 4 telescopes does not provide a good IPR graph. Is that case caught somewhere?

Reply:
Fixed by replacing 4 by getNTel() in all occurrences. Introduced new parameter to check the norm of the sum of all IPR graphs. In case this is lower than 100000 (set by parameter fPedPerTelescopeTypeMinCnt), the user receives anwarning.